test/unit/deadline_job_attachments/test_asset_sync.py

```diff
@@ -465,6 +486,78 @@ def test_sync_inputs_with_step_dependencies_same_root_vfs_on_posix(
                 },
             ]
 
+    @pytest.mark.parametrize(
+        ("job_fixture_name"),
+        [
+            ("default_job"),
+        ],
+    )
+    @pytest.mark.parametrize(
+        ("s3_settings_fixture_name"),
+        [
+            ("default_job_attachment_s3_settings"),
+        ],
+    )
+    def test_sync_inputs_no_space_left(
```

Comment:
Should we add a test case that this doesn't raise an error if VFS is being used?

Reply:
Sure yea I can do that

Reply:
Oh wait, there's no need, I forgot I'm doing an `assert_not_called` in this VFS test, which is doing the test you suggested: https://github.com/aws-deadline/deadline-cloud/blob/ed9539a02474183b17bb214afb013ccaaf7d8153/test/unit/deadline_job_attachments/test_asset_sync.py#L474